packages/grafana-ui/src/components/Drawer/Drawer.tsx

```diff
@@ -56,35 +60,31 @@ export function Drawer({
   tabs,
 }: Props) {
   const styles = useStyles2(getStyles);
-  const [isExpanded, setIsExpanded] = useState(false);
-  const [isOpen, setIsOpen] = useState(false);
   const overlayRef = React.useRef(null);
   const { dialogProps, titleProps } = useDialog({}, overlayRef);
   const { overlayProps } = useOverlay(
     {
       isDismissable: false,
-      isOpen,
+      isOpen: true,
```

Comment:
Why can we now set this to `true` always?

Reply:
@JoaoSilvaGrafana 

because the animation did not work in rc-drawer version 4 but seems fixed in the version we use now (v6)

Reply:
I mean it did not work when we set isOpen to true on mount so we had to default to open false, and then change it to true on mount to get the animation to work, but works now without that

Reply:
ah nice! makes sense